packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -545,6 +546,8 @@ abstract class PageTransitionsBuilder {
 ///    that's similar to the one provided in Android Q.
 ///  * [CupertinoPageTransitionsBuilder], which defines a horizontal page
 ///    transition that matches native iOS page transitions.
+///  * [PredictiveBackPageTransitionsBuilder], which defines a page
+///    transition that allows peeking behind the current route on Android.
```

Comment:
nit: Should this documentation also include the Android version supported by predictive back, similar to how the documentation for the other TransitionsBuilders in this list denote the Android version? Here and in other references.